src/uagents/network.py

```diff
@@ -24,7 +144,9 @@
 else:
     raise NotImplementedError
 
-_contract = LedgerContract(None, _ledger, CONTRACT_ALMANAC)
+
+_almanac_contract = AlmanacContract(None, _ledger, CONTRACT_ALMANAC)
+_service_contract = ServiceContract(None, _ledger, CONTRACT_SERVICE)
```

Comment:
Probably more clear to call this `_name_service_contract`, `NameServiceContract`.